main/include/main.h

```diff
@@ -0,0 +1,59 @@
+
+
+#ifndef MAIN_H_
+#define MAIN_H_
+
+#include <esp_wifi.h>
+#include <esp_event.h>
+#include <esp_log.h>
+#include <esp_system.h>
+#include <nvs_flash.h>
+#include <sys/param.h>
+#include "protocol_examples_common.h"
+#include "SpotifyAuthorization.h"
+#include "HttpsRequest_.h"
+#include "WiFiConfig_.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/event_groups.h"
+#include "freertos/queue.h"
+#include "nvsFlash.h"
+
+#define MYSSID "Hardware12"
```

Comment:
every config should define inside of its component.

Reply:
ok